pages/docs/reference/delegated-properties.md

```diff
@@ -444,4 +487,21 @@ class C {
 
 </div>
 
-Note that the `provideDelegate` method affects only the creation of the auxiliary property and doesn't affect the code generated for getter or setter.
+Note that the `provideDelegate` method affects only the creation of the auxiliary property and doesn't affect the code
+generated for getter or setter.
+
+With the `PropertyDelegateProvider` interface from the standard library, you can create delegate providers as anonymous
+objects without creating new classes.
+
+<div class="sample" markdown="1" theme="idea" data-highlight-only auto-indent="false">
+
+```kotlin
+fun makeDelegate() = PropertyDelegateProvider<Nothing?, Int> =
```

Comment:
`<Nothing?, Int>` — that seems to be a quite strange combination of type arguments.

- `Nothing?` for the property owner type limits delegation to top-level properties only. If the property owner type doesn't matter, better to specify it as `Any?`.
- `Int`: here should be the delegate object type, not the property type. While it's possible to define delegation operators for `Int` as extensions, it's usually not the case.